### PR TITLE
Add native support for MultiDensityImageSource

### PR DESCRIPTION
**Pull request by kristianhasselknippe**
Meaning it can be used within a NativeViewHost.

This PR contains:
- [x] Changelog
- [ ] Documentation
- [ ] Tests

### CODE REVIEW

File: Source/Fuse.Controls.Native/Android/ImageView.uno [hunk 1]
```diff
@@ -72,22 +72,34 @@ namespace Fuse.Controls.Native.Android
...
+			{
+				return _imageSource;
+			}
+		}
+
+		void OnImageSourceChanged()
+		{
+			SourceChanged;
+
+				var mds = (MultiDensityImageSource)ImageSource;
```
Comment (mortoray) [inline Source/Fuse.Controls.Native/Android/ImageView.uno]: Soemthing with the spacing is wrong here.

Comment (mortoray) [reply Source/Fuse.Controls.Native/Android/ImageView.uno]: There's a chunk of code missing too it seems, when compared to the iOS code.

File: Source/Fuse.Controls.Native/iOS/ImageView.uno [hunk 2]
```diff
@@ -13,22 +13,44 @@ namespace Fuse.Controls.Native.iOS
...
+		{
+			if (ImageSource == null)
+				return;
+			else if (ImageSource is FileImageSource)
+				UpdateImage((FileImageSource)ImageSource);
+			else if (ImageSource is HttpImageSource)
+				UpdateImage((HttpImageSource)ImageSource);
+			else if (ImageSource is MultiDensityImageSource)
+			{
+				if (ImageSource != null && ImageSource is MultiDensityImageSource)
```
Comment (mortoray) [inline Source/Fuse.Controls.Native/iOS/ImageView.uno]: This cleanup won't happen if the user switches from a MultiDensityImageSource to a different type of ImageSource.
A common pattern in fuselibs is on settting a new value, cleanup the old one first, regardless of what it is, then handle the new value.  You've kind of mixed the two steps together here.

File: Source/Fuse.Elements/Resources/MultiDensityImageSource.uno [hunk 3]
```diff
@@ -92,6 +110,7 @@ namespace Fuse.Resources
 			}
 
 			SwapActive(use);
+			OnSourcesChanged();
```
Comment (mortoray) [inline Source/Fuse.Elements/Resources/MultiDensityImageSource.uno]: The name of the event seems wrong, shouldn't it be `OnActiveChanged`?  It probably belongs in the `SwapActive` function.

File: Source/Fuse.Controls.Native/Android/ImageView.uno (hunk 1 above)
Comment (kristianhasselknippe) [reply Source/Fuse.Controls.Native/Android/ImageView.uno]: yeah, no idea how i missed that, but that's indeed what happened :S

**Pull request merged**